test/regress/cli/run_regression.py

```diff
@@ -536,20 +536,45 @@ def run_benchmark(benchmark_info):
     )
 
     # If a scrubber command has been specified then apply it to the output.
+    scrubber_error = ""
     if benchmark_info.scrubber:
-        output, _, _ = run_process(
+        output, scrubber_error, _ = run_process(
             benchmark_info.scrubber,
             benchmark_info.benchmark_dir,
             benchmark_info.timeout,
             output,
         )
+    # Make sure that the scrubber itself did not print anything to its error output
+    if len(scrubber_error) != 0:
+        print_error("The scrubber's error output not empty")
+        print()
+        print("  Command: {}".format(benchmark_info.scrubber))
+        print()
+        print("  Error output")
+        print("  " + "=" * 78)
+        print(scrubber_error)
+        print("  " + "=" * 78)
+        return EXIT_FAILURE
```

Comment:
It might be a good idea to put this check into a function so that we can use it below and don't need to duplicate the code block. Something like `check_scrubber(scrubber_error, benchmark_info.scrubber)`.